lldb/source/Plugins/ExpressionParser/Clang/ClangExpressionParser.cpp

```diff
@@ -576,7 +644,8 @@ ClangExpressionParser::ClangExpressionParser(
     lang_opts.GNUMode = true;
     lang_opts.GNUKeywords = true;
     lang_opts.CPlusPlus11 = true;
-    lang_opts.BuiltinHeadersInSystemModules = true;
+    lang_opts.BuiltinHeadersInSystemModules = !sdkSupportsBuiltinModules(
```

Comment:
This `!` is the intended direction, right?

Reply:
Yes. The builtin headers have to go in the system modules if the sdk doesn't support the builtin modules.